src/Microsoft.TemplateEngine.Cli/Installer.cs

```diff
@@ -237,5 +265,21 @@ private void InstallLocalPackages(IReadOnlyList<string> packageNames)
                 }
             }
         }
+
+
+        internal virtual bool ExecuteProcess(string command, params string[] args)
```

Comment:
Well I went back and found 2 options in Dotnet.cs and ProcessStartPostActionProcessor.cs.  Neither of these appear to be generic implementations that I could use from the Installer class.  Is there another?